src/index.html

```diff
@@ -22,6 +22,418 @@
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__content">
+          <img
+            class="card__photo"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          />
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__code">Product code: 195434</div>
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            Reviews: 5
+          </div>
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+
+            <div class="card__price-tag">$2,199</div>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__content">
+          <img
+            class="card__photo"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          />
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__code">Product code: 195434</div>
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            Reviews: 5
+          </div>
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+
+            <div class="card__price-tag">$2,199</div>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="hover"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+      <div
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div
```